rmf_fleet_adapter_python/CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+## Changelog for package rmf_fleet_adapter_python
+
+1.3.0 (2021-06-07)
+------------------
+* Modifications to support refactor of rmf_task (`#51 <https://github.com/open-rmf/rmf_ros2/issues/51>`_)
```

Comment:
Don't we normally link PRs in the changelogs and not issues?

Reply:
This was actually brought in from the current `main` branch, so it should be addressed in a separate PR to avoid merge conflicts.

That said, the link seems to work anyway since GitHub uses unique integers across both issues and PRs, and their server is smart enough to redirect.